lib/App/DuckPAN/Cmd/New.pm

```diff
@@ -130,8 +130,9 @@ sub run {
 	# Check which IA repo we're in...
 	my $type = $self->app->get_ia_type();
 
+	my $no_handler = 0;
 	# Process the --cheatsheet option
-	if ($self->cheatsheet) {
+	if ($self->cheatsheet || $self->template && $self->template eq "cheatsheet") {
```

Comment:
@zachthompson during testing I was using `duckpan new --template cheatsheet` and it wasn't being caught by this.

I've improved this check so it catches that and now everything works as expected